### PR TITLE
Add fix for fullstackopen code comment highlights

### PR DESCRIPTION
**Pull request by jeansibelius**
Define a higher contrast background color for the code comments in fullstackopen.com when using the dynamic mode of Dark reader.

Example can be viewed for example here: https://fullstackopen.com/en/part8/fragments_and_subscriptions#subscriptions-on-the-client

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -5724,6 +5724,14 @@ img[alt="[NLUUG]"]
 
 ================================
 
+fullstackopen.com
+
+CSS
+.gatsby-highlight-code-line {
+    background-color: rgba(77, 80, 83, 0.3);
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5731]: So what about:
```suggestion
    background-color: ${hsla(0,0%,48.2%,.3)} !important;
```

Comment (jeansibelius) [reply src/config/dynamic-theme-fixes.config:5731]: Hi! Thanks a lot for the review and suggestion. If it's ok to ask a couple of questions, I'm curious why hsla over rgba? And the other one: why is the expression wrapped in `${ ... }`? Is that for a preprocessor used in this project?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:5731]: > , I'm curious why hsla over rgba?

It's what the CSS(from fullstackopen) used, I only modified 88.2% -> 48.2% to make it more highlighting

> why is the expression wrapped in `${ ... }`? Is that for a preprocessor used in this project?

https://github.com/darkreader/darkreader/blob/master/CONTRIBUTING.md#how-to-use-the-dev-tools -> Look at the Rules table

**Review by Gusted — APPROVED** [review]

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -5724,6 +5724,14 @@ img[alt="[NLUUG]"]
...
 ================================
 
+fullstackopen.com
+
+CSS
+.gatsby-highlight-code-line {
+    background-color: ${hsla(0,0%,48.2%,.3)} !important;
+}
+
+================================
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5734]:
```suggestion
================================

```

**Pull request merged**